neo/SmartContract/InteropService.cs

```diff
@@ -336,31 +308,31 @@ private static bool Blockchain_GetBlock(ApplicationEngine engine)
                 hash = new UInt256(data);
             else
                 return false;
-            if (hash == null)
-            {
+
+            Block block = hash != null ? engine.Snapshot.GetBlock(hash) : null;
+            if (block == null)
                 engine.CurrentContext.EvaluationStack.Push(new byte[0]);
```

Comment:
We should wailt for https://github.com/neo-project/neo-vm/pull/208

Reply:
Merged